app/models/doctor_time.rb

```diff
@@ -0,0 +1,4 @@
+class DoctorTime < ApplicationRecord
+  belongs_to :doctor
+  has_many :time_schedules
```

Comment:
Please ensure that the models have proper validation to prevent the storage of inaccurate data in the database.